UnityProject/Assets/Scripts/Player/LightEmissionPlayer.cs

```diff
@@ -97,6 +97,11 @@ public void UpdatelightSource()
 			//CurrentLight not updated yet
 			return;
 		}
+		if(lightSprite == null)
+		{
+			Logger.LogWarning("lightSprite is null (probably blank field in inspector)", Category.Lighting);
+			return;
+		}
 		lightSprite.Color = CurrentLight.Colour;
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
LightEmissionPlayer.UpdatelightSource () (at /github/workspace/UnityProject/Assets/Scripts/Player/LightEmissionPlayer.cs:100)
LightEmissionPlayer.Awake () (at /github/workspace/UnityProject/Assets/Scripts/Player/LightEmissionPlayer.cs:130)
UnityEngine.Object:Instantiate(GameObject, Vector3, Quaternion, Transform)
ObjectPool:TryPoolInstantiate(GameObject, SpawnDestination, Boolean, GameObject&) (at /github/workspace/UnityProject/Assets/Scripts/Lifecycle/ObjectPool.cs:197)
SpawnablePrefab:SpawnAt(SpawnDestination) (at /github/workspace/UnityProject/Assets/Scripts/Lifecycle/Spawnable/SpawnablePrefab.cs:55)
Spawn:Server(SpawnInfo) (at /github/workspace/UnityProject/Assets/Scripts/Lifecycle/Spawn.cs:278)
Spawn:ServerPrefab(GameObject, Nullable`1, Transform, Nullable`1, Int32, Nullable`1, Boolean, Boolean) (at /github/workspace/UnityProject/Assets/Scripts/Lifecycle/Spawn.cs:133)
LivingHealthBehaviour:Harvest() (at /github/workspace/UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs:883)
<>c__DisplayClass11_0:<ServerPerformInteraction>g__ProgressFinishAction|0() (at /github/workspace/UnityProject/Assets/Scripts/Weapons/Meleeable.cs:146)
StandardProgressAction:OnServerEndProgress(EndProgressInfo) (at /github/workspace/UnityProject/Assets/Scripts/UI/ProgressBar/StandardProgressAction.cs:245)
ProgressBar:UpdateMe() (at /github/workspace/UnityProject/Assets/Scripts/UI/ProgressBar/ProgressBar.cs:245)
UpdateManager:Update() (at /github/workspace/UnityProject/Assets/Scripts/Managers/UpdateManager/UpdateManager.cs:253)